public_html/dataView/allfiles.php

```diff
@@ -41,6 +41,8 @@
 	}
 
 	//determine the field name
+	// FIXME: this will only grab the LAST file field, if a form has Multiple
```

Comment:
This appears incomplete, if there is a FIXME in the comments your acknowledging that the code doesn't work properly.  These are the things I want to avoid merging into the codebase.